components/widgets/users/avatar/avatar.tsx

```diff
@@ -11,6 +11,26 @@ import BotDefaultIcon from 'images/bot_default_icon.png';
 
 export type TAvatarSizeToken = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
 
+export const getAvatarWidth = (size: TAvatarSizeToken) => {
```

Comment:
I needed this to style buttons containing avatars, so they would be the right size. Let me know if there's a preferable way to doing this, though.